### ISSUE TITLE
[Improvement]: Implement a separate docker compose file for codeception for more clarity

### ISSUE THREAD

**Issue by aryaantony92** [pimcore/skeleton#141]
### Improvement description

Follow up to #109

**Comment by yariksheptykin** [comment]
@aryaantony92 could you describe a bit more what you aim for with this improvement? Maybe I can do a PR if you find it helpful.

**Comment by yariksheptykin** [comment]
Do you what to move [these two services](https://github.com/pimcore/skeleton/blob/11.x/docker-compose.yaml#L66-L89) into a separate docker-compose.yml? Like docker-compose.test.yml?

**Comment by aryaantony92** [comment]
> @aryaantony92 could you describe a bit more what you aim for with this improvement? Maybe I can do a PR if you find it helpful.

@yariksheptykin Sure, Thank you!

**Comment by aryaantony92** [comment]
> Do you what to move [these two services](https://github.com/pimcore/skeleton/blob/11.x/docker-compose.yaml#L66-L89) into a separate docker-compose.yml? Like docker-compose.test.yml?

@yariksheptykin Yes exactly. Also, please update the steps in readme that for running codeception, we need to bring up the container with the new docker-composer-test.yaml and then run the codeception command.
